app/models/application_record.rb

```diff
@@ -1,6 +1,8 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
 
+  include Purgable
```

Comment:
Previously, the `purge` behavior was on all models so I'm doing the same thing by including it here.

Reply:
I think this should be call `Purgeable` 😊 see https://www.merriam-webster.com/dictionary/purgeable

Reply:
Oopsie! Good catch 🙈